bundles/GraphicServiceOrder/Controller/GraphicServiceOrderController.php

```diff
@@ -28,7 +28,7 @@ class GraphicServiceOrderController extends AbstractController
     /**
      * Create a service order.
      *
-     * @Route("/", name="form")
+     * @Route("/create-graphic-service-order", name="form")
```

Comment:
You should move this prefix to the `@Route` annotation on the controller class itself (cf. https://symfony.com/blog/new-in-symfony-3-4-prefix-all-controller-route-names).

Reply:
The showDeliveryNote function needs to start with /jira to fall under the the right firewall. So either that would force an expansion of the firewall setup or a separate controller for the delivery note alone. So why not just keep it?

Reply:
Ok. Good point.